controllers/idler/idler_controller_test.go

```diff
@@ -256,6 +262,116 @@ func TestEnsureIdling(t *testing.T) {
 			})
 		})
 	})
+
+	t.Run("Create notification the first time resources Idled", func(t *testing.T) {
+		// given
+		idler := &toolchainv1alpha1.Idler{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "alex-stage",
+				Labels: map[string]string{
+					toolchainv1alpha1.OwnerLabelKey: "alex",
+				},
+			},
+			Spec: toolchainv1alpha1.IdlerSpec{TimeoutSeconds: 60},
+		}
+		namespaces := []string{"dev", "stage"}
+		usernames := []string{"alex"}
+		nsTmplSet := newNSTmplSet(nstemplatesetTest.MemberOperatorNS, "alex", "advanced", "abcde11", namespaces, usernames)
+		mur := newMUR("alex")
+		reconciler, req, cl, allCl := prepareReconcile(t, idler.Name, newGetHostClusterReady, idler, nsTmplSet, mur)
+		idlerTimeoutPlusOneSecondAgo := time.Now().Add(-time.Duration(idler.Spec.TimeoutSeconds+1) * time.Second)
+		podsRunningForTooLong := preparePayloads(t, reconciler, idler.Name, "todelete-", idlerTimeoutPlusOneSecondAgo)
+
+		// first reconcile to track pods
+		res, err := reconciler.Reconcile(context.TODO(), req)
+		require.NoError(t, err)
+		assert.True(t, res.Requeue)
+		assert.Equal(t, int(res.RequeueAfter), 0)
+		// Idler tracks all pods now but pods have not been deleted yet
+		memberoperatortest.AssertThatInIdleableCluster(t, allCl).
+			PodsExist(podsRunningForTooLong.standalonePods).
+			DaemonSetExists(podsRunningForTooLong.daemonSet).
+			JobExists(podsRunningForTooLong.job).
+			DeploymentScaledUp(podsRunningForTooLong.deployment).
+			ReplicaSetScaledUp(podsRunningForTooLong.replicaSet).
+			DeploymentConfigScaledUp(podsRunningForTooLong.deploymentConfig).
+			ReplicationControllerScaledUp(podsRunningForTooLong.replicationController).
+			StatefulSetScaledUp(podsRunningForTooLong.statefulSet)
```

Comment:
I think we can remove this since we're not really testing this behaviour in this testcase and it's covered by the other unit tests. Removing it would reduce noise and keep focus on what we really want to test.
```suggestion
```